config/seata-config-custom/src/test/java/io/seata/config/CustomConfigurationForTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+
+public class CustomConfigurationForTest extends AbstractConfiguration<ConfigChangeListener> {
+    private Properties properties;
+
+    public CustomConfigurationForTest(String name) {
+        try {
+            try (InputStream input = CustomConfigurationForTest.class.getClassLoader().getResourceAsStream(name)) {
+                properties = new Properties();
+                properties.load(input);
+            }
```

Comment:
Why not use try with resource-catch directly?

Reply:
Didn't pay much attention to this test class. You are right, it's much better to use try with resource-catch directly. I'll fix it.

Reply:
Committed.